### DISCUSSION TITLE
How to begin C# development on Linux?

### DISCUSSION THREAD

**Discussion by gitowiec** [consulo/consulo#447] [Q&A] [answered]
I use Ubuntu 18.04 LTS as my devel. platform. I would like to try to code in .NET Core with C#. I was looking for any tutorial on how to begin to code with Consulo using C# but I can't find any. I read all wiki. I found Unity "how-to", I am going to try it, but I wish I could also code in C# without Unity. How can I start?
(To clarify, I don't know how to start with .NET Core and C# without Consulo either, I just wanted to begin a whole new experience with this IDE, because on daily basis I work with Webstorm so I thought I could give it a try - I'm familiar with the look and feel of the Jetbrains IDEs)

**Comment by VISTALL** [comment, marked as answer]
Hello. I will upset you. .NET Core(and now .NET 5) is not supported. There only few (now) request at current moment (https://github.com/consulo/incubating-consulo-dotnet-core/issues/5) 

Sorry, and thanks. I will up priority in my mind)
